cypress/scripts/run-starter-prototype.js

```diff
@@ -1,18 +1,24 @@
 const os = require('os')
 const path = require('path')
 
-const { mkPrototype, startPrototype, installExtensions } = require('../../__tests__/util')
+const { mkPrototype, startPrototype, installExtensions, npmInstall } = require('../../__tests__/util')
 
 const defaultKitPath = path.join(os.tmpdir(), 'cypress/temp/test-project')
 
 const testDir = path.resolve(process.env.KIT_TEST_DIR || defaultKitPath)
 
 ;(async () => {
   await mkPrototype(testDir, { overwrite: true, allowTracking: false })
+  const fooLocation = path.join(__dirname, '..', 'fixtures', 'extensions', 'extension-foo')
+  const barLocation = path.join(__dirname, '..', 'fixtures', 'extensions', 'extension-bar')
+  await Promise.all([
+    npmInstall(fooLocation),
+    npmInstall(barLocation)
+  ])
```

Comment:
Why is this using `Promise.all`? `npmInstall` is a sync function.

Reply:
I had a version where it was async but it got lost when I was sorting out the history.  I've made the function async because I want these two to happen at the same time to avoid increasing the test time unnecessarily.

Reply:
Ah, I see. Can that work? npm won't complain about multiple processes trying to change the same dependency tree?